scripts/limonade/src/index.ts

```diff
@@ -57,16 +59,52 @@ const argv = yargs(hideBin(process.argv))
     type: "number",
     default: 30 * 1000,
   })
+  .option("hermes-streaming-timeout", {
+    description: "Hermes streaming timeout (milliseconds)",
+    type: "number",
+    default: 5 * 1000,
+  })
+  .option("price-config", {
+    description:
+      "Path to the price config file, if not provided, we will not get price info from Hermes",
+    type: "string",
+  })
+  .option("hermes-endpoint", {
+    description: "Hermes endpoint to use for fetching price updates",
+    type: "string",
+    default: "https://hermes.pyth.network/",
+  })
+  .option("off-market-threshold", {
+    description:
+      "Threshold of price ratio to consider an opportunity off-market",
+    type: "number",
+    default: 1.05,
```

Comment:
can you add a parameter for how often we try to resubmit opportunities?
This parameter and the frequency go hand in hand. We can deploy multiple limonade instances and tune these. For example, one that resubmits stuff faster if they are closer to the current price

Reply:
done